snap/snapcraft.yaml

```diff
@@ -362,8 +364,8 @@ parts:
       - libvulkan1:amd64
       - libxml2:i386
       - libxml2:amd64
-      - libicu70:i386
-      - libicu70:amd64
+      - libicu74:i386
+      - libicu74:amd64
```

Comment:
Can't comment on unchanged lines, but `changelog.all.gz` doesn't exist, needs to be `changelog.*` below.